zulipterminal/model.py

```diff
@@ -333,6 +333,22 @@ def mark_message_ids_as_read(self, id_list: List[int]) -> None:
         })
         display_error_if_present(response, self.controller)
 
+    @asynch
+    def send_typing_event(self, recipient_user_ids: List[int],
+                          typing_status: bool) -> None:
```

Comment:
That sounds appropriate, I'll rename them and modify the function as well.

Reply:
This looks much clearer :+1:

Reply:
@neiljp do you mean the method's name - `send_typing_status_by_user_ids`? I have also reduced the redundancy in the request dictionary by using the status directly since we're only accepting `start` and `stop`. Do you think it needs further refinement?